docs/freqai-running.md

```diff
@@ -0,0 +1,139 @@
+
+Trained models are by default saved to disk to allow for reuse during backtesting or after a crash. The user can opt to purge old models to save disk space by setting `"purge_old_models": true` in the config.
+
+If the user wishes to start a dry/live run from a saved backtest model (or from a previously crashed dry/live session), the user only needs to specify the `identifier` of the specific model:
+
+```json
+    "freqai": {
+        "identifier": "example",
+        "live_retrain_hours": 0.5
+    }
+```
+
+In this case, although FreqAI will initiate with a pre-trained model, it will still check to see how much time has elapsed since the model was trained. If a full `live_retrain_hours` has elapsed since the end of the loaded model, FreqAI will start training a new model.
+
+## Backtesting
+
+The FreqAI backtesting module can be executed with the following command:
+
+```bash
+freqtrade backtesting --strategy FreqaiExampleStrategy --strategy-path freqtrade/templates --config config_examples/config_freqai.example.json --freqaimodel LightGBMRegressor --timerange 20210501-20210701
+```
+
+If this command has never been executed with the existing config file, FreqAI will train a new model
+for each pair, for each backtesting window within the expanded `--timerange`.
+
+Backtesting mode requires the user [download the necessary data](freqai-data-handling.md#downloading-data-to-cover-the-full-backtest-period) before deployment (unlike in dry/live mode where FreqAI handles the data downloading automatically). The user should be careful to consider that the time range of the downloaded data is more than the backtesting time range. This is because FreqAI needs data prior to the desired backtesting time range in order to train a model to be ready to make predictions on the first candle of the user-set backtesting time range. More details on how to calculate the data to download can be found [here](#deciding-the-size-of-the-sliding-training-window-and-backtesting-duration). 
+
+!!! Note "Model reuse"
+    Once the training is completed, the user can execute the backtesting again with the same config file and
+    FreqAI will find the trained models and load them instead of spending time training. This is useful
+    if the user wants to tweak (or even hyperopt) buy and sell criteria inside the strategy. If the user
+    *wants* to retrain a new model with the same config file, then they should simply change the `identifier`.
+    This way, the user can return to using any model they wish by simply specifying the `identifier`.
+
+---
```

Comment:
Above here should be a good place for the "Downloading data"  part - that'll avoid having the user skip around.